src/styles/_catalog.scss

```
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color 300ms;
+    transition: transform 300ms;
```

value 300ms is the same for all transitions, use var

```suggestion
    transition: color 300ms, transform 300ms;
```